src/domain/profile/profile.dto.ts

```diff
@@ -1,16 +1,13 @@
-import { InputType, Field } from '@nestjs/graphql';
-import { MaxLength } from 'class-validator';
```

Comment:
am guessing we want to leave these then in given the discussion this morning?

Reply:
Only for the description. The avatar is url. The url by specification can be up to 2MB. Which is not very likely to happen.

Reply:
But we can add constrain IsUri